frontend/src/components/AccountReceiveTable.vue

```diff
@@ -119,7 +119,15 @@
               </q-card-section>
```

Comment:
*(arbitrary comment location)*

Looks fine, and all functionality seems to work OK. One issue I did encounter. If you:

1. Send base asset
2. Withdraw to the stealth address (ignoring the warning)

Then upon refresh, that transaction is still shown as not withdrawn. Attempting withdrawal at that point through the interface results in a JSON RPC error.

This does not happen with token withdrawals to the stealth address, which properly show as withdrawn.

Ideally, we would show the txn as withdrawn even if the user withdrew to the same stealth address.

Reply:
> The JSON RPC error is odd though, what is the specific error? Wondering if we should also pull that into a separate issue since it's not relevant for this hotfix and likely a rare case

<img width="754" alt="Screen Shot 2022-03-30 at 11 52 21 AM" src="https://user-images.githubusercontent.com/1479802/160877745-d0128233-fcb2-4878-b8ce-2de729528f11.png">

This makes sense, since the stealth address already paid some gas for the "withdrawal" tx to itself, the calculation no longer works correctly.

> For ETH/native tokens we show that it's withdrawn if stealth address balance is less than the amount sent.

This seems not to be the case, or to have a bug of some kind. If it were working, the above address would have shown as withdrawn, because it had paid for gas and thus had a slightly smaller balance. Also, in one case, I actually imported the private key and withdrew ~all of the MATIC (metamask left some dust) but that case *also* shows as not-withdrawn in the interface.

Reply:
Got it, what's the current algo— checks for 0 balance?